website/content/docs/api-gateway/usage/basic-usage.mdx

```diff
@@ -0,0 +1,135 @@
+---
+layout: docs
+page_title: Consul API Gateway Basic Usage
+description:  >-
+  Consul API Gateway Basic Usage
+---
+
+
+# Basic Usage
+
+This topic describes the basic workflow for implementing Consul API Gateway configurations.
+
+1. Verify that the [requirements](/docs/api-gateway/consul-api-gateway-install#requirments) have been met.
+1. Verify that the Consul API Gateway CRDs and controller have been installed and applied (see [Installation](/docs/api-gateway/consul-api-gateway-install)).
+1. Configure the artifacts as describe in the [Configuration](/docs/api-gateway/configuration) section.
+
+<CodeBlockConfig hideClipboard filename="values.yaml">
+
+  ```yaml
+  apiGateway:
+    enabled: true
+    managedGatewayClass:
+  ```
+
+</CodeBlockConfig>
```

Comment:
is this the wrong place? this doesn't feel right. also, can we provide a high level working example? the next command references `gateway.yaml` and `routes.yaml`, neither of which was defined

ideally, we can replace this page with a "Get Started" that links out to the API Gateway learn tutorial (https://learn.hashicorp.com/tutorials/consul/kubernetes-api-gateway). what do you think @trujillo-adam?